src/Components/Web.JS/src/Boot.Server.ts

```diff
@@ -37,13 +37,8 @@ async function boot(userOptions?: Partial<BlazorOptions>): Promise<void> {
     }
 
     const reconnection = existingConnection || await initializeConnection(options, logger);
-    if (reconnection.state !== signalR.HubConnectionState.Connected) {
-      logger.log(LogLevel.Information, 'Reconnection attempt failed. Unable to connect to the server.');
-      return false;
-    }
-
     if (!(await circuit.reconnect(reconnection))) {
-      logger.log(LogLevel.Information, 'Reconnection attempt to the circuit failed.');
+      logger.log(LogLevel.Information, 'Reconnection attempt to the circuit refused.');
```

Comment:
instead of `refused` could we say something like `Reconnection attempt to the circuit was rejected by the server. This may indicate that the associated state is no longer available on the server.`

Reply:
I kinda like the term `rejected` better than `refused` for this use case. Thoughts?

Reply:
> Reconnection attempt to the circuit was rejected by the server. This may indicate that the associated state is no longer available on the server.

Sounds great!

> I kinda like the term `rejected` better than `refused` for this use case. Thoughts?

Definitely considered rejected as well. It was my 2nd choice. Because of my indifference I rolled the die and picked `refused`. We can move to team `rejected` 😉